analytics_dashboard/settings/base.py

```diff
@@ -416,7 +416,7 @@
 
 ########## THEME CONFIGURATION
 # Path of the SCSS file to use for the site's theme
-THEME_SCSS = 'sass/themes/open-edx.scss'
+THEME_SCSS = os.getenv('THEME_SCSS', 'sass/themes/open-edx.scss')
```

Comment:
@OmarIthawi We don't need this change, the configuration playbooks are including the `INSIGHTS_THEME_SCSS` in the insights yaml config (`/edx/etc/insights.yml`). https://github.com/appsembler/configuration/blob/appsembler/ficus/master/playbooks/roles/insights/defaults/main.yml#L125

For more details, the production settings loads the env in this way:
```
from analytics_dashboard.settings.base import *
from analytics_dashboard.settings.yaml_config import *
from analytics_dashboard.settings.logger import get_logger_config
```
So `THEME_SCSS` is going to be replaced anyway.

Reply:
Yeah, we don't. I didn't see those imports before!